tests/test_boolean.phpt

```diff
@@ -0,0 +1,61 @@
+--TEST--
+IBM-DB2: Boolean data type test.
+--SKIPIF--
+<?php
+
+require_once('skipif.inc');
+
+// This test requires IBM i 7.5 or a version of LUW that supports boolean
+require_once('connection.inc');
+$conn = db2_connect($database, $user, $password);
+if (!$conn) {
+	die("skip can't connect to DB:" . db2_conn_errormsg());
+}
+
+$info = db2_server_info($conn);
+if ($info->DBMS_NAME == "AS") { // IBM i
+	// DBMS_VER is VVRRM string
+	$major = $info[1];
+	$minor = $info[3];
+	$mod = $info[4];
+	if (!version_compare("$major.$minor.$mod", "7.5.0", ">=")) {
+		die("skip IBM i version too old");
+	}
+}
+// XXX: LUW, z
```

Comment:
Looks like support was added to LUW in 9.7: https://www.ibm.com/docs/en/db2/9.7?topic=supported-boolean-data-type

Probably should add that at least to facilitate CI runs.